sandbox/cri/config.toml

```diff
@@ -0,0 +1,189 @@
+# Firecracker
+[proxy_plugins]
+  [proxy_plugins.firecracker-naive]
+    type = "snapshot"
+    address = "/var/run/firecracker-containerd/snapshotter.sock"
```

Comment:
You can't have both firecracker-naive and firecracker-dm-snapshotter using the same socket address.

(Also, I'd probably use "firecracker-naive" and "firecracker-dm" to be consistent with each other.)